tests/robot-tests/tests/public_api/public_api_major_auto_changes.robot

```diff
@@ -0,0 +1,253 @@
+*** Settings ***
+Library             ../libs/admin_api.py
+Resource            ../libs/admin-common.robot
+Resource            ../libs/admin/manage-content-common.robot
+Resource            ../libs/public-common.robot
+Resource            ../libs/public-api-common.robot
+
+Force Tags          Admin    Local    Dev    AltersData
+
+Suite Setup         user signs in as bau1
+Suite Teardown      user closes the browser
+Test Setup          fail test fast if required
+Test Teardown       Run Keyword If Test Failed    record test failure
+
+
+*** Variables ***
+${PUBLICATION_NAME}=    UI tests - Public API - major auto changes %{RUN_IDENTIFIER}
+${RELEASE_1_NAME}=      Financial year 3000-01
+${RELEASE_2_NAME}=      Academic year 3010/11
+${SUBJECT_1_NAME}=      UI tests - Public API - major auto changes - subject 1 - %{RUN_IDENTIFIER}
+${SUBJECT_2_NAME}=      UI tests - Public API - major auto changes - subject 2 - %{RUN_IDENTIFIER}
+
+
+*** Test Cases ***
+Create publication and release
+    ${PUBLICATION_ID}=    user creates test publication via api    ${PUBLICATION_NAME}
+    user creates test release via api    ${PUBLICATION_ID}    FY    3000
+    user navigates to draft release page from dashboard    ${PUBLICATION_NAME}
+    ...    ${RELEASE_1_NAME}
+
+Verify release summary
+    user checks page contains element    xpath://li/a[text()="Summary" and contains(@aria-current, 'page')]
+    user verifies release summary    Financial year    3000-01    Accredited official statistics
+
+Upload datafile
+    user uploads subject and waits until complete    ${SUBJECT_1_NAME}    absence_school.csv    absence_school.meta.csv
+    ...    ${PUBLIC_API_FILES_DIR}
+
+Add data guidance to subjects
+    user clicks link    Data and files
+    user waits until h2 is visible    Add data file to release
+
+    user clicks link    Data guidance
+    user waits until h2 is visible    Public data guidance
+
+    user waits until page contains element    id:dataGuidance-dataFiles
+    user waits until page contains accordion section    ${SUBJECT_1_NAME}
+
+    user enters text into data guidance data file content editor    ${SUBJECT_1_NAME}
+    ...    ${SUBJECT_1_NAME} Main guidance content
+
+    user clicks button    Save guidance
+
+Create 1st API dataset
```

Comment:
Think this should be `Create API data set` as we only create a single API data set in this test suite (and then update it)

Reply:
See prev comment.  Now "initial data set version".